requirements/docs.txt

```diff
@@ -10,3 +10,4 @@ matplotlib>=3.0.1
 dask[array]>=0.15.0
 # cloudpickle is necessary to provide the 'processes' scheduler for dask
 cloudpickle>=0.2.1
+pandas
```

Comment:
Some reasonable lower bound for the version would be nice. As a reference, I suggest to use NEP29 (see https://github.com/scikit-image/scikit-image/issues/4371).